.mvn/wrapper/maven-wrapper.properties

```diff
@@ -1 +1,14 @@
+#Copyright 2021-2023 Ness Digital Engineering
```

Comment:
Please return back formatting to way how it was before with using
You should only adjust TEXT not Format !